modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/VariantUtils.cs

```diff
@@ -486,7 +486,7 @@ public static godot_signal ConvertToSignal(in godot_variant p_var)
             => NativeFuncs.godotsharp_variant_as_signal(p_var);
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static SignalInfo ConvertToSignalInfo(in godot_variant p_var)
+        public static Signal ConvertToSignalManaged(in godot_variant p_var)
```

Comment:
nitpick

```suggestion
        public static Signal ConvertToManagedSignal(in godot_variant p_var)
```

I'm open for better naming suggestions, though. I understand we don't go with `SignalObject` because it's a struct. Maybe `ConvertToSignalWrapper`.

Reply:
I'm just following the naming of `ConvertToCallableManaged`, should that be changed too?

Reply:
Oh, I didn't see that one. Yes, if we're going to rename one, then the other one as well for consistency. But if it's too much, we can keep the current naming, as it's just a nitpick.

Honestly, I'm not a big fan of "managed". I wish I had gone with `ConvertToNativeX` and `ConvertToX` instead.

Reply:
I went ahead and merged as is, I think the change to `ConvertTo{Native,}X` for both `Callable` and `Signal` can be done as a follow-up.